docs/06-concepts/06-database/01-connection.md

```diff
@@ -39,26 +45,64 @@ development:
 
 ## Development database
 
-A newly created Serverpod project comes with a preconfigured docker instance with Postgresql. All you have to do is run the following command from the root of the `server` package to start the database.
-
-:::info
-
-[Docker](https://www.docker.com/) is required for this to work.
-
-:::
+A newly created Serverpod project has a preconfigured Docker instance with a Postgres database set up. Run the following command from the root of the `server` package to start the database:
 
 ```bash
 $ docker compose up --build --detach
 ```
 
-To remove the database run (this commands preserve all data).
+To stop the database run:
 
 ```bash
-$ docker compose down
+$ docker compose stop
 ```
 
-To remove the database and __delete__ all data add the `-v` flag.
+To remove the database and __delete__ all associated data, run:
 
 ```bash
 $ docker compose down -v
 ```
+
+## Connecting to a custom Postgres instance
+
+Just like you can connect to the Postgres database inside the Docker container, you can connect to any other Postgres instance. There are a few things you need to take into consideration:
+
+- Make sure that your Postgres instance allows incoming traffic.
+- You will need to create a user with a password, and a database.
+
+### Connecting to a local Postgres server
+
+If you want to connect to a local Postgres Server (with the default setup) then the `development.yaml` will work fine if you set the correct port, user, database, and update the password in the `passwords.yaml` file.
+
+### Connecting to a remote Postgres server
+
+To connect to a remote Postgres server (that you have installed on a VPS or VDS), you need to follow a couple of steps:
+
+- Make sure that the Postgres server has a reachable network address and that it accepts incoming traffic.
+- You may need to open the database port on the machine. This may include configuring its firewall.
+- Configure Postgres to listen to connections from your Serverpod server.
```

Comment:
Suggestion: This is a bit vague as well and I wouldn't know how to resolve it.

Could you clarify what it means by maybe linking some instructions or giving an example?